modules/ROOT/pages/install-port-reqs.adoc

```diff
@@ -41,6 +41,7 @@ The following table lists the TCP ports that must be accessible:
 | 53 | TCP | Internal | Internal DNS
 | 7496 | TCP | Internal  | Peer-to-peer connectivity
 | 5000 | TCP | Internal | Docker registry
+| 5044 | TCP | Outbound to internet | Anypoint Monitoring, Anypoint Visualizer
```

Comment:
I think we should make the "Outbound to internet" ports more visible so they don't get overlooked. Perhaps we move this line so it's adjacent to the first one?

Reply:
Thanks!